Feature carousel

soltykiewiczm opened the pull request:
carousel ready
awaiting comments or acceptance

soltykiewiczm committed:
initial game html outline concept

soltykiewiczm committed:
added carousel in the header in index.html
added header styling and media in style.css

soltykiewiczm committed:
changed position of glypicons in the carousel

maciekmp reviewed (changes requested):
wygląd slidera nie powala, nie jest na całą szerokość, proszę o użycie domyślnych strzałek lewo-prawo i usunięcie dodatkowych styli związanych z nadpisaniem
PR zawiera kod związany z grą którego tutaj nie powinno być

skwirowski reviewed (commented):
Hej, to moje uwagi:
1. karuzela jest za nisko powinieneś usunąć padding-top 90px, żeby zaczynała się zaraz pod paskiem menu,
2. Rozumiem co miałeś na myśli wrzucając pierwsze zdjęcie, ale moim zdaniem jest ono odrzucające, zdjęcie nr 2 wygląda moim zdaniem tak sobie, zdjęcie nr 3 bardzo mi się podoba (aż ślinka cieknie).
3. Moja wskazówka jest taka, żeby dać ładne zdjęcie i dopiero wymyślić do niego jakiś tekst, przykładowo:
a) takie zdjęcie: https://i.pinimg.com/736x/fe/7a/14/fe7a147b77ba7e2960d6708e9178b8e5--fridge-organization-ideas-refrigerator-organization.jpg
i napis "masz pełną lodówkę, ale zero pomysłów co ugotować? nie mogłeś trafić lepiej!"
b) takie zdjęcie:
https://static7.depositphotos.com/1288156/783/i/950/depositphotos_7831999-stock-photo-empty-fridge.jpg
i napis "masz pustą lodówke? To nie problem, razem z nami przygotujesz kurczaka w pięciu smakach". Jakbyś zostawił w którymś slajdzie te jabłka, które były wcześniej, to by się fajnie komponowało.
4. Karuzela nie jest na 100% viewportu.
5. Podstawowe (domyślne) elementy karuzeli (strzałki) lepiej się prezentują.
6. Docelowo przydałoby się więcej zdjęć/slajdów. Wydaje mi się, że wtedy łatwiej byłoby przyciągnąć użytkowników (dużo zdjęć pysznego jedzenia i śmiesznych komentarzy :) )

To chyba wszystko,

Pozdrawiam

soltykiewiczm committed:
changed control icons
changed pictures and captions
added caption background to make captions more visible and readable
removed padding-top
fit to full screen

karoob commented:
Dalej jest coś nie tak z obrazkami w karuzeli.
1. Na mobile: między 320 a 337px podjeżdża za wysoko. Jest to pewnie spowodowane tym że przycisk menu spada na dół bo obrazek z logiem w nav jest za duży. Spróbuję się tym zająć dzisiaj w osobnym branchu także tym bym się nie przejmował za bardzo obecnie.
2. Na desktopach obrazki nie są na środku i do tego nie zajmują wraz z navbar'em 100% wysokości i/lub szerokości.
3. Na mobile mogłaby być wyższa i również zajmować z navbarem cała wysokość.
PS 4. Skoro staramy się trzymać clean-code to karuzele też powinieneś formatować podobnie jak robisz w clean-code branchu bo przy przy połączeniu do developa ten branch nie będzie trzymał standardu. Chodzi o atrybuty w 'span', 'img' i 'a'. 

Tyle ode mnie.

filipburakowski reviewed (commented):
Dalej jest coś nie tak z obrazkami w karuzeli.
1. Na mobile: między 320 a 337px podjeżdża za wysoko. Jest to pewnie spowodowane tym że przycisk menu spada na dół bo obrazek z logiem w nav jest za duży. Już zająłem się tym w osobnym branchu także nie przejmuj się tym na razie.
2. Na desktopach obrazki nie są na środku i do tego nie zajmują wraz z navbar'em 100% wysokości i/lub szerokości.
3. Na mobile mogłaby być wyższa i również zajmować z navbarem cała wysokość.
PS 4. Skoro staramy się trzymać clean-code to karuzele też powinieneś formatować podobnie jak robisz w clean-code branchu bo przy przy połączeniu do developa ten branch nie będzie trzymał standardu. Chodzi o atrybuty w 'span', 'img' i 'a'. 

Tyle ode mnie.

filipburakowski committed:
Fixing carousel to fit 100% of screen. Added extra class for carousel items and added CSS.

soltykiewiczm committed:
changed caption styling
increased font-size for mobiles in captions

soltykiewiczm committed:
erased unnecesary media in style.css
